content/blog/using-apache-spark-dataframes-for-processing-of-tabular-data.md

```diff
@@ -0,0 +1,362 @@
+---
+title: Using Apache Spark DataFrames for Processing of Tabular Data
+date: 2021-10-14T05:35:43.497Z
+priority: 4
+author: Carol McDonald
+authorimage: /img/Avatar1.svg
+tags:
+  - HPE Ezmeral Data Fabric
```

Comment:
Replace the tag "HPE Ezmeral Data Fabric" with "hpe-ezmeral-data-fabric" to link the post to the Platform "HPE Ezmeral Data Fabric".

Reply:
done